src/gnatss/ops/posfilter.py

```diff
@@ -1,11 +1,15 @@
 from typing import List
 
+import numpy as np
 from nptyping import Float, NDArray, Shape
 from pandas import DataFrame
 from pymap3d import ecef2enu
 from scipy.spatial.transform import Rotation
 
+from gnatss import constants
 from gnatss.configs.solver import ArrayCenter
+from gnatss.loaders import load_gps_solutions
+from gnatss.ops.kalman import run_filter_simulation

```

Comment:
Imports should be relative within a package in case package name changes. Please modify to be relative.